commit b6e580702666134ba64a49f3fccaf6edefca44e6
Author: Andrew Johnson <anj@aps.anl.gov>
Date:   Fri Nov 4 17:18:14 2016 -0500

    Add .local includes to makeBaseApp templates
    
    (cherry picked from commit aa6e976f92d144b5143cf267d8b2781d3ec8b62b)

diff --git a/src/template/base/top/configure/CONFIG_SITE b/src/template/base/top/configure/CONFIG_SITE
index 208aa7e68..d78c7f514 100644
--- a/src/template/base/top/configure/CONFIG_SITE
+++ b/src/template/base/top/configure/CONFIG_SITE
@@ -34,3 +34,9 @@ CHECK_RELEASE = YES
 #   or CROSS_OPT settings from base/configure/CONFIG_SITE
 #HOST_OPT = NO
 #CROSS_OPT = NO
+
+# These allow developers to override the CONFIG_SITE variable
+# settings without having to modify the configure/CONFIG_SITE
+# file itself.
+-include $(TOP)/../CONFIG_SITE.local
+-include $(TOP)/configure/CONFIG_SITE.local
diff --git a/src/template/base/top/configure/RELEASE b/src/template/base/top/configure/RELEASE
index 84074fe15..79ef8dee7 100644
--- a/src/template/base/top/configure/RELEASE
+++ b/src/template/base/top/configure/RELEASE
@@ -30,3 +30,8 @@ EPICS_BASE=_EPICS_BASE_
 # Set RULES here if you want to take build rules from somewhere
 # other than EPICS_BASE:
 #RULES=/path/to/epics/support/module/rules/x-y
+
+# These allow developers to override the RELEASE variable settings
+# without having to modify the configure/RELEASE file itself.
+-include $(TOP)/../RELEASE.local
+-include $(TOP)/configure/RELEASE.local
